addons/hr_attendance/migrations/12.0.2.0/post-migration.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_hr_employee_last_attendance_id(env):
```

Comment:
The same logic for this: as the SQL column doesn't exist, regular update process creates it and compute the value.